components/patient-data/rest/src/main/java/org/phenotips/data/rest/internal/DefaultPatientsResourceImpl.java

```diff
@@ -93,34 +95,95 @@
     private Provider<Autolinker> autolinker;
 
     @Override
-    public Response addPatient(String json)
-    {
+    public final Response add(final String json) {
```

Comment:
Not our [codestyle](https://github.com/phenotips/build-tools/tree/master/codestyle-resources/src/main/resources).